generators/Exercises/Bowling.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Text;
+using Generators.Input;
+using Generators.Output;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Generators.Exercises
+{
+    public class Bowling : Exercise
+    {
+        private const string PreviousRolls = "previous_rolls";
+
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in canonicalData.Cases)
+            {
+                if (!(canonicalDataCase.Expected is long))
+                {
+                    canonicalDataCase.Expected = null;
+                }
+                canonicalDataCase.SetInputParameters();
+                canonicalDataCase.UseVariableForTested = true;
+                canonicalDataCase.Property = "score";
```

Comment:
When we switch to using exceptions, this line should be removed as the actual property being tested is then not `score`, but `roll`.